api/filters.go

```diff
@@ -779,11 +775,11 @@ func setErrorCode(w http.ResponseWriter, err error, typ ...string) {
 	case "Instance not found":
```

Comment:
Should this error message be changed from instance to version?

Reply:
Done